x-pack/plugins/cases/server/client/attachments/update.ts

```diff
@@ -39,7 +40,7 @@ export async function update(
       ...queryRestAttributes
     } = queryParams;
 
-    decodeCommentRequest(queryRestAttributes);
+    decodeCommentRequest(queryRestAttributes, externalReferenceAttachmentTypeRegistry);
```

Comment:
Makes sense to have it here but in practice, we don't need it right? We don't have tests for it either.

Reply:
Yeah I don't think the user would update the files through the UI. We don't prevent a user from hitting the update API to change the contents of an attachment framework attachment though so I'll add a test for it.